cms/djangoapps/contentstore/views/component.py

```diff
@@ -181,11 +181,11 @@ def container_handler(request, usage_key_string):
                 parent = get_parent_xblock(parent)
             ancestor_xblocks.reverse()
 
-            assert unit is not None, "Could not determine unit page"
+            assert unit is not None, _("Could not determine unit page")
             subsection = get_parent_xblock(unit)
-            assert subsection is not None, "Could not determine parent subsection from unit " + unicode(unit.location)
+            assert subsection is not None, _("Could not determine parent subsection from unit %s") % unicode(unit.location)  # pylint: disable=line-too-long
             section = get_parent_xblock(subsection)
-            assert section is not None, "Could not determine ancestor section from unit " + unicode(unit.location)
+            assert section is not None, _("Could not determine ancestor section from unit %s") % unicode(unit.location)
```

Comment:
Please use `.format` and named variables; see http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#use-named-placeholders